app/workers/fetch_graph_worker.rb

```diff
@@ -23,7 +23,12 @@ def perform(pid)
       work.attributes[controlled_prop.to_s].each do |val|
         # Fetch labels
         if val.respond_to?(:fetch)
-          val.fetch(headers: { 'Accept' => default_accept_header })
+          begin
+            val.fetch(headers: { 'Accept' => default_accept_header })
+          rescue TriplestoreAdapter::TriplestoreException
+            fetch_failed_graph(pid, user)
```

Comment:
Should an email go out for individual failures like previously?
`Hyrax.config.callback.run(:ld_fetch_error, user, pid)`